suite/tests/api/ir_regdeps.c

```diff
@@ -209,13 +250,16 @@ test_instr_create_encode_decode_synthetic_x86_64(void *dc)
     instr_encode(dc, instr, buf);
     instr_reset(dc, instr);
     decode(dc, buf, instr);
-    test_instr_encode_decode_synthetic(dc, instr);
+    const char *expected_disasm_str_vdpbf16ps_mask =
+        " 00000031 65646640 uncategorized [64byte]       %rv98 %rv99 %rv102 -> %rv100\n "
+        "00000068";
+    test_instr_encode_decode_synthetic(dc, instr, expected_disasm_str_vdpbf16ps_mask);
 }
 #endif
 
 #ifdef ARM
```

Comment:
Both ARM and RISCV64 tests build but are not being run (looking at Github tests: "ci-aarchxx-cross / arm-cross-compile" and "ci-riscv64 / riscv64-cross-compile" respectively).
Are my define wrong? (But I believe those are what's used in other tests as well)
Or am I doing something wrong in CMakeLists.txt that I'm not seeing?

https://github.com/DynamoRIO/dynamorio/blob/a559edc0b4c6a2ac4c2c94702a6a7bf87e408533/suite/tests/CMakeLists.txt#L2040

Reply:
The cross runs only execute a short list of tests known to succeed under QEMU.  Look for the RUNS_ON_QEMU test lists at the bottom of the file.

Reply:
Ah, that's what I was missing, thanks.

Reply:
I enabled the `api.ir_regdeps` test for APPLE (still runs on x86_64, so no changes needed) and RISCV64 (categories there are all "uncategorized", but better than nothing).
Couldn't enable it for ARM, as I get: "/lib/ld-linux-armhf.so.3: No such file or directory", so that's disabled (added a FIXME comment about it, if we want to enable it in the future).